content/kubeone/master/infrastructure/production_recommendations.en.md

```diff
@@ -4,4 +4,61 @@ date = 2020-04-01T12:00:00+02:00
 weight = 4
 +++
 
-TBD
\ No newline at end of file
+## AWS
+
+### ami_id
+
+{{% notice warning %}}
+It's very important to set this terraform variable to avoid future terraform attempts to recreate your control plane
+instance.
+{{% /notice %}}
+
+Can be set AFTER initial `terraform apply`, and you can find its initial discovered values in your terraform state by
+using:
+
+```bash
+terraform state show data.aws_ami.ami
+# data.aws_ami.ami:
+data "aws_ami" "ami" {
+    ...
+    id                    = "ami-00f6fb16625871821"
+    ...
+```
+
+This this example AMI ID is `"ami-00f6fb16625871821"`, in your case it may be different. 
+
+In terraform.tfvars file:
+```terraform
+ami_id = "ami-00f6fb16625871821"
+```
+
+### internal_api_lb
+
+In order to hide your kubernetes API endpoint from the external world, it's recommended to use `internal_api_lb` which
+will cause ELB to be created in "internal" mode (accessible only from inside of your VPC).
+
+```terraform
+internal_api_lb = true
+```
+
+In order to access your cluster later from outside, there is build-in HTTPS proxy tunnel in KubeOne.
+
+```bash
+kubeone proxy -t .
+SSH tunnel started, please open another terminal and setup environment
+export HTTPS_PROXY=http://127.0.0.1:8888
```

Comment:
Split into two blocks.

Reply:
This is direct output of the command, why split?